src/core/unit.c

```diff
@@ -2589,13 +2506,13 @@ static int unit_log_resources(Unit *u) {
                         r = log_oom();
                         goto finish;
                 }
-
                 joined[0] = ascii_toupper(joined[0]);
+
                 t = strjoina("MESSAGE=", u->id, ": ", joined, ".");
```

Comment:
this might grow quite large, maybe we shouldn't use stack allocation anymore here, i.e. use strjoin() rather than strjoina()